recipes/googletest/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "1.7.0" %}
+{% if linux %}
+    {% set libext = ".so" %}
+{% else %}
+    {% set libext = ".dylib" %}
+{% endif %}
+
+package:
+  name: googletest
+  version: {{ version }}
+
+source:
+  fn: release-{{ version }}.tar.gz
+  url: https://github.com/google/googletest/archive/release-{{ version }}.tar.gz
+  sha256: f73a6546fdf9fce9ff93a5015e0333a8af3062a152a9ad6bcb772c96687016cc
+
+build:
+  number: 0
+  skip: true  # [win or py3k]
+
+requirements:
+  build:
+    - cmake
+    - toolchain
```

Comment:
We had `python` before, but we seem to have lost it. Should we be adding it back?